File: tags/rel-2-2-2/bindings/php5/examples/get_attributes_from_assertion.php
/* Example SP PHP5 code to get attributes from an assertion */

foreach ($assertion->attributeStatement[0]->attribute as $attribute) {
    if ($attribute->name == LASSO_SAML2_ATTRIBUTE_NAME_EPR) {
        continue;
    }
    echo 'attribute : ' . $attribute->name . "\n";
    foreach ($attribute->attributeValue as $value) {
        echo '  value : ' . $value->any[0]->content . "\n";
    }
}

File: trunk/bindings/python/examples/get_attributes_from_assertion.py
# Example SP Python code to get attributes from an assertion

for attribute in assertion.attributeStatement[0].attribute:
    if attribute.name == lasso.SAML2_ATTRIBUTE_NAME_EPR:
        continue
    print 'attribute : ' + attribute.name
    for value in attribute.attributeValue:
        print '  value : ' + value.any[0].content
